modules/test/playwright/tests/frontend-data-set-views-web/itemActions.spec.ts

```diff
@@ -0,0 +1,65 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {featureFlagsTest} from '../../fixtures/featureFlagsTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {actionsPageTest} from './fixtures/actionsPageTest';
+import {dataSetsPageTest} from './fixtures/dataSetsPageTest';
+import {viewsPageTest} from './fixtures/viewsPageTest';
+
+export const test = mergeTests(
+	actionsPageTest,
+	dataSetsPageTest,
+	featureFlagsTest({
+		'LPS-164563': true,
+		'LPS-194395': true,
+	}),
+	loginTest,
+	viewsPageTest
+);
+
+const LINK_ITEM_ACTION_NAME = 'Link item action';
+
+test('Create a Link Item Action', async ({
+	actionsPage,
+	dataSetsPage,
+	page,
+	viewsPage,
+}) => {
+	await test.step('Create Data Set', async () => {
+		await dataSetsPage.goto();
+		await dataSetsPage.createSampleDataSet();
```

Comment:
Do we clean this at the end of tests? I see this is now created in both test files.

Would it maybe make sense to create a `dataSetsPage.deleteSampleDataSet()`, and run it as the last step?

cc @dsanz @chestofwonders

Reply:
Juanjo fixed this in a later commit :)